### PR TITLE
Add over18 use case to demo website

### PR DESCRIPTION
**Pull request by rafaelbernard**
https://app.asana.com/0/1171124854073033/1199407824084301

- Create index page to summarize
- Move old index to Simple issuer page
- Create wine shop page

### CODE REVIEW

File: src/election.html [hunk 1]
```diff
@@ -16,7 +16,7 @@
   <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
 
   <script src="assets/irma.js" defer></script>
-  <script src="irma-functions.js" defer></script>
+  <script src="assets/js/irma-functions.js" defer></script>
```
Comment (ehsan-fj) [inline src/election.html:19]: Mm... I think assets are like external stuff!

Comment (rafaelbernard) [reply src/election.html:19]: Can be a way of facing it. 
I then should move css/ and apk/ out of assets folder, right?

Comment (ehsan-fj) [reply src/election.html:19]: Yup that's another issue. Keep doing what your doing now. It's a demo site and really doesn't matter.

**Review by ehsan-fj — APPROVED** [review]
Well done! 🚀

File: src/election.html (hunk 1 above)
Comment (rafaelbernard) [reply src/election.html:19]: Deal!

It was good to bring the discussion and we can agree with that when the time comes. ;-)

**Pull request merged**